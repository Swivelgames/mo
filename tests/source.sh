#!/usr/bin/env bash

cd "${0%/*}" || exit 1
cat <<EOF | ../mo --source=source.vars
{{#BASH}}Purposely Unsafe for Backwards Compatibility{{/BASH}}
{{VAR}}
{{#ARR}}
* {{.}}
{{/ARR}}
{{ASSOC_ARR.a}} {{ASSOC_ARR.b}}
EOF
