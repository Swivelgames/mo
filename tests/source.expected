Purposely Unsafe for Backwards Compatibility
value
* 1
* 2
* 3
AAA BBB
